### PR TITLE
New quirk: Handy

### PR DESCRIPTION
**Pull request by Mickyan**
:cl: Mickyan
add: New quirk: Handy. Can you fix it? Yes you can! You can fix simple structures using a wrench instead of a welding tool.
/:cl:
You start with a wrench which you can use it to repair damaged windows, plating and walls instead of using a welding tool.
The wrench you get has some custom text just for extra flavor and because I like the idea of having little clues like these, when applicable, for other people to notice and find out you have a certain quirk. 
Otherwise it's just a regular wrench, you can just throw it away if you like.

This is just gives a little extra quality of life if you want to do repair jobs around the workplace without the need to search around the station for a welder and eye protection, you'll still need materials as usual for repairing breaches.

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
on the one hand muh realisms but on the other hand this is a really cool little thing that is what traits should be about

**Comment by Mickyan** [comment]
I hear ya but it also makes as much sense as being able to shape metal into rods or building walls with our bare hands

I think it's the kind of thing that isn't so absurd we can't just rely on suspension of disbelief for the sake of gameplay

**Comment by ninjanomnom** [comment]
Done properly this should really be a refactor to how things are considered tools and the quirk would either modify wrenches you pick up, or, tools when checking what they can be used for would check in with the user for modifications.

That's a fairly massive refactor though for this, I just dread more features similar to this coming up and making that inevitable refactor a massive pain to do.

**Comment by Mickyan** [comment]
@ninjanomnom I had the same thoughts to be honest, if maintainers decide this isn't worth the future headaches I totally understand

**Comment by Razharas** [comment]
Snowflaky checks for types of tool and presence of a trait are horrible
Either remake or dont touch it

**Comment by europax** [comment]
I don't like this PR, to be honest. It literally makes eye damage from lack of welding helmet useless.

**Comment by Mickyan** [comment]
not even close

literally not

**Comment by ExcessiveUseOfCobblestone** [comment]
@Repukan @europax It still gives the eye damage.

Basically if you have this trait any wrench is a welding tool too.

Not a fan at all of the implementation. The concept of being a tool needs to be component.

**Comment by Mickyan** [comment]
@ExcessiveUseOfCobblestone That's not right, you can repair walls, windows and plating with a wrench. That's all there is to it. It doesn't give eye damage but the dozens of other more important actions that require welders still do as usual.

**Comment by ExcessiveUseOfCobblestone** [comment]
Even better! Still, implementation is meh

**Comment by Bawhoppen** [comment]
Neat.

**Comment by optimumtact** [comment]
why do they start with a wrench?

**Comment by Mickyan** [comment]
I already touched on that in the description but It's already situational as it is, if you have to stop by tool storage every round to actually get some use out of it you may as well save a point and just grab a welder or just deconstruct the thing you want to repair and rebuild it.

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -181,19 +181,20 @@
...
-			to_chat(user, "<span class='notice'>You begin repairing [src]...</span>")
-			if(I.use_tool(src, user, 40, volume=50))
-				obj_integrity = max_integrity
-				update_nearby_icons()
-				to_chat(user, "<span class='notice'>You repair [src].</span>")
-		else
-			to_chat(user, "<span class='warning'>[src] is already in good condition!</span>")
-		return
+	if(user.a_intent == INTENT_HELP)
+		if(istype(I, /obj/item/weldingtool) || (user.has_trait(TRAIT_HANDY) && (istype(I, /obj/item/wrench))))
```
Comment (Jalleo) [inline code/game/objects/structures/window.dm]: How does this react with the deconstruction of windows?

Comment (Mickyan) [reply code/game/objects/structures/window.dm]: Oof, good catch I can't believe I forgot to test that. Can't think of a way to fix it that isn't even more hacky so I think this is dead for now.

**Pull request closed without merging**